frontend/src/app/members/add/[...groupID]/page.tsx

```diff
@@ -2,11 +2,21 @@ import AddMembers from '@/components/AddMembers'
 import Navbar from '@/components/Navbar'
 import React from 'react'
 
-export default function page() {
+export default function Page() {
   return (
-    <div>
-      <Navbar/>
-      <AddMembers/>
+    <div className="h-screen">
```

Comment:
Structure of all these modifications is similar. Extract them in a component which takes other components as props